pmm-app/tests/verifyPMMSettingsPageElements_test.js

```diff
@@ -166,5 +145,48 @@ xScenario(
 
     await pmmSettingsPage.waitForPmmSettingsPageLoaded();
     I.waitForValue(pmmSettingsPage.fields.dataRetentionCount, dataRetention, 30);
-  }
+  },
+);
+
+Scenario(
+  'PMM-T415 - Verify Percona Platform (Sign up) elements on PMM Settings Page @not-pr-pipeline',
+  async (I, pmmSettingsPage) => {
+    const label = 'Check here to indicate that you have read and agree to the \nTerms of Service\n and \nPrivacy Policy';
```

Comment:
All text is shown in settings Page is already available in page class above the fields, please check, I think it could be included in there.